cmd/feed-ingress/main.go

```diff
@@ -338,6 +347,17 @@ func createIngressUpdaters() ([]controller.Updater, error) {
 		}
 		updaters = append(updaters, elbUpdater)
 
+		statusConfig := elb_status.Config{
+			Region:           region,
+			LabelValue:       elbLabelValue,
+			KubernetesClient: controllerConfig.KubernetesClient,
```

Comment:
I think it would be better to pass the client into this function I think from main(). Rather than rely on the controller config, which may/may not change independently.